app/components/page-components/Chatbot/windows/ChatbotCommandWindow.vue

```diff
@@ -0,0 +1,71 @@
+<template>
+<ModalLayout
+  :showControls="false"
+  :customControls="true"
+  :title="$t('Add Command')"
+>
+  <div slot="fixed">
+    <Tabs :tabs="tabs" :value="selectedTab" @input="onSelectTab">
+    </Tabs>
+  </div>
+  <div slot="content" class="chatbot-add-command__container">
+    <div v-if="selectedTab === 'general'">
+      <div>
+        <label for="command" class="margin-vertical--10">Command</label>
+        <WTextInput
+          v-model="newCommand.command"
+          :metadata="commandMetadata"
+        />
+      </div>
+      <div>
+        <label for="response" class="margin-vertical--10">Response</label>
+        <WTextAreaInput
+          v-model="newCommand.response"
+          :metadata="responseMetadata"
+        />
+      </div>
+      <div>
+        <div class="small-6 columns">
+          <label for="permission" class="margin-vertical--10">Permission</label>
+          <WListInput
+            v-model="newCommand.permission.level"
+            :metadata="permissionMetadata"
+          />
+        </div>
+        <div class="small-6 columns">
+          <label for="show to" class="margin-vertical--10">Show to</label>
+          <WListInput
+            v-model="newCommand.response_type"
+            :metadata="showToMetadata"
+          />
+        </div>
+      </div>
+    </div>
+    <div v-if="selectedTab === 'advanced'">
+      Advanced Stuff
+    </div>
+  </div>
+  <div slot="controls">
+    <button
+      class="button button--default"
+      @click="onCancel">
+      Cancel
+    </button>
+    <button
+      class="button button--action"
+      @click="onSave"
+    >
+      <!-- <i class="fa fa-spinner fa-pulse" v-if="updatingInfo" /> -->
+      Save
```

Comment:
We should take a pass through all of these components before this feature is complete to make sure we get all of these strings in the translation system.

Reply:
For now, we only need to wrap it `$t()` function.  So we can find all strings to translate with a regexp in the future.